sql/analyzer/triggers.go

```diff
@@ -355,49 +354,6 @@ func orderTriggersAndReverseAfter(triggers []*plan.CreateTrigger) []*plan.Create
 	return append(beforeTriggers, afterTriggers...)
 }
 
-func OrderTriggers(triggers []*plan.CreateTrigger) (beforeTriggers []*plan.CreateTrigger, afterTriggers []*plan.CreateTrigger) {
```

Comment:
You don't want to move this, I don't think. It's not DDL, it's about where to insert the trigger in the plan (before and after others)

Reply:
Yea i was problems with import loops. I don't think this method should be in the analyzer

Reply:
It definitely belongs in the analyzer, where else would it go?